src/components/surveys/survey-design/question-edit/rhs-subcontrols/Year.test.tsx

```diff
@@ -46,7 +46,7 @@ function setUp(step: YearQuestion) {
 
 const setUpWithChangeMock = (step: YearQuestion) => {
   const onChangeMock = jest.fn()
-  const user = userEvent.setup()
+  const user = userEvent.setup({advanceTimers: jest.advanceTimersByTime})
```

Comment:
We're using fakeTimers, so advance timers when testing-library internally [delays](https://testing-library.com/docs/user-event/options/#delay) subsequent code.